clusterloader2/pkg/measurement/util/gatherers/container_resource_gatherer.go

```diff
@@ -0,0 +1,232 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Copied from k8s.io/kubernetes/test/e2e/framework/resource_usage_gatherer.go
```

Comment:
Sorry - I didn't mean to add a comment in the file - I meant github comment to make review easier.
Please remove it, because it will probably soon be outdated.

And everywhere else.

Reply:
Comments removed.